versions/3.0.md

```diff
@@ -763,7 +763,7 @@ Describes a single API operation on a path.
 Field Name | Type | Description
 ---|:---:|---
 <a name="operationTags"></a>tags | [`string`] | A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
-<a name="operationSummary"></a>summary | `string` | A short summary of what the operation does. For maximum readability in editing or documentation generation tools, this field SHOULD be less than 120 characters.
```

Comment:
@earth2marsh we did discuss clarifying the difference between `summary` and `description`. I don't think we should mention tooling here, just clarify the difference in goals.